magento/pwa-studio
# 4.0.0 changelog

## Description

Changelog for the 4.0.0 release

## Related Issue

Closes #1583

## Verification Steps

1. View `CHANGELOG.md`
2. Verify items are accurate for the 4.0.0 release

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
